src/styles/components/_details.scss

```diff
@@ -39,15 +40,21 @@ $BORDER_COLOR: $GREY_300;
     outline: none;
   }
 
-  &:hover .w-details__header::after,
-  &:focus .w-details__header::after {
-    background: $LIGHT_BLUE_50;
+  @include hover() {
+    &:hover .w-details__header::after,
+    &:focus .w-details__header::after {
+      background: $LIGHT_BLUE_50;
+    }
   }
 
   &:focus .w-details__header::after {
     box-shadow: 0 0 0 1px $FOCUS_COLOR;
   }
 
+  .js-focus-visible &:focus:not(.focus-visible) .w-details__header::after { // sass-lint:disable-line class-name-format
```

Comment:
Do you want to add the `.js-focus-visible` thing to other places where focus-visible is used in this PR?

Reply:
It wasn't working when used with web components (Tabs and CopyCode). Wasn't sure why. Help appreciated!

Reply:
Did you try:

```
.js-focus-visible .web-code-copy ...
```

?
```

Reply:
Fixed. I apparently can't read nested rules... <sigh>